atd-etl/data_model/utils/graphql.py

```diff
@@ -0,0 +1,180 @@
+from datetime import datetime, timezone
+import os
+
+import requests
+
+from utils.exceptions import HasuraAPIError
+
+ENDPOINT = os.getenv("HASURA_GRAPHQL_ENDPOINT")
+ADMIN_SECRET = os.getenv("HASURA_GRAPHQL_ADMIN_SECRET")
+
+COLUMN_METADATA_QUERY = """
+query ColumnMetadata {
```

Comment:
this `_column_metadata` table is sooo clutch im so glad we are doing this! reminding me of our recent convos on documentation, would be good to have a protocol written up somewhere on adding/deleting columns that lists the steps, ie. having to add new columns to this table. also is there a constraint to make sure all columns in the tables are in the column metadata table?